core/camera/src/main/java/com/google/jetpackcamera/core/camera/CameraXCameraUseCase.kt

```diff
@@ -428,6 +428,14 @@ constructor(
         )
     }
 
+    override suspend fun pauseVideoRecording() {
+        videoCaptureControlEvents.trySendBlocking(VideoCaptureControlEvent.PauseRecordingEvent)
+    }
+
+    override suspend fun resumeVideoRecording() {
+        videoCaptureControlEvents.trySendBlocking(VideoCaptureControlEvent.ResumeRecordingEvent)
+    }
+
     override fun stopVideoRecording() {
         videoCaptureControlEvents.trySendBlocking(VideoCaptureControlEvent.StopRecordingEvent)
```

Comment:
Just realized we're using `trySendBlocking` here. `stopVideoRecording` should be converted to a `suspend` fun and `trySendBlocking` should be `send`.